app/lib/actions/katello/system/bulk_auto_attach_subscriptions.rb

```diff
@@ -0,0 +1,34 @@
+module Actions
+  module Katello
+    module System
+      class BulkAutoAttachSubscriptions < Actions::EntryAction
```

Comment:
'Bulk' tends to imply doing this operation to one or more, in this case I would expect this to allow me to auto attach subscriptions to multiple systems?